src/it/java/com/google/checkstyle/test/chapter2filebasic/rule21filename/OuterTypeFilenameTest.java

```diff
@@ -21,7 +21,7 @@
 
 import static com.puppycrawl.tools.checkstyle.checks.OuterTypeFilenameCheck.MSG_KEY;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
```

Comment:
If the old class is forbidden, I would add that to the import control to ban.

Reply:
done